packages/dumb/src/components/stats/insulin/insulin-stat.tsx

```diff
@@ -0,0 +1,205 @@
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import React, { type FunctionComponent, memo } from 'react'
+import styles from './insulin-stat.css'
+import Box from '@mui/material/Box'
+import Chip from '@mui/material/Chip'
+import { t } from 'i18next'
+import { formatDecimalNumber } from '../../../utils/format/format.util'
+import { StatTooltip } from '../../tooltips/stat-tooltip/stat-tooltip'
+import { useLocation } from 'react-router-dom'
+import { EMPTY_DATA_PLACEHOLDER } from '../../../models/stats.model'
+import { useTheme } from '@mui/material/styles'
+
+interface TotalInsulinPropsData {
+  id: string
+  title: string
+  units: string
+  value: number
+  valueString: string
+}
+
+export interface TotalInsulinStatProps {
+  data: TotalInsulinPropsData[]
+  totalInsulin: number
+  weight: number | string
+  dailyDose: number
+}
+
+const InsulinStat: FunctionComponent<TotalInsulinStatProps> = (props) => {
+  const {
+    data,
+    totalInsulin,
+    dailyDose,
+    weight
+  } = props
+  const theme = useTheme()
+  const location = useLocation()
+  const isDailyPage = location.pathname.includes('daily')
+  const isDashboardPage = location.pathname.includes('dashboard')
+  const isTrendsPage = location.pathname.includes('trends')
+  const annotations = [t(isDailyPage ? 'total-insulin-days-tooltip' : 'average-daily-insulin-tooltip'), t('total-insulin-how-calculate-tooltip')]
+  const isDisabledWeight = weight === EMPTY_DATA_PLACEHOLDER
+  if (data.length === 0) {
+    annotations.push(t('tooltip-empty-stat'))
+  }
+  const dailyDosePerWeight = (): string | number => {
+    if (weight === EMPTY_DATA_PLACEHOLDER) {
+      return EMPTY_DATA_PLACEHOLDER
+    }
+    const value = dailyDose / +weight
+    return value > 0 && Number.isFinite(value) ? formatDecimalNumber(value, 2) : EMPTY_DATA_PLACEHOLDER
+  }
+  const isDailyDosePerWeight = dailyDosePerWeight.toString() === EMPTY_DATA_PLACEHOLDER
+
+  const getOutPutValueClasses = (): string => {
+    return `${isDailyDosePerWeight ? `${styles.outputValueDisabled}` : `${styles.dailyDoseValue}`}`
+  }
+
+  const getPercentage = (value: number): string => {
+    const res = Math.round(100 * value / totalInsulin)
+    return res > 0 ? res.toString(10) : EMPTY_DATA_PLACEHOLDER
+  }
+
+  return (
+    <div data-testid="container-insulin-stats">
+      <Box>
+        <Box
+          display="flex"
+          justifyContent="space-between"
+          alignItems="center"
+          marginBottom="4px"
```

Comment:
useless, and not in harmony with MUI spacing rules, as we discussed, as far as possible, you should use the theme.spacing method. In this case this margin bottom is useless !